### ISSUE TITLE
Restore backsups - provide sort on the date

### ISSUE THREAD

**Issue by derwin12** [xLightsSequencer/xLights#5062]
It appears the restore back UI just dumps all the backups - can this be sorted and/or provide a column header to sort.
![image](https://github.com/user-attachments/assets/b4cf9687-1231-4791-8327-eba0ea7f4312)

**Comment by cybercop23** [comment]
It'd needs to be number aware sort too

**Comment by cybercop23** [comment]
So this seems to be Mac only. Same show/backup folder using Mac vs PC has a different sort. Windows is already sorted, Mac is random. Can you please confirm you see the same on your Windows setup, @derwin12?

**Comment by derwin12** [comment]
Yes, windows somehow gets it right .. but I figure we could use the sort option anyways .. asc/desc.

**Comment by derwin12** [comment]
I believe Scott addressed this..
